pkg/component/component.go

```diff
@@ -44,14 +46,15 @@ func CreateFromGit(client *occlient.Client, name string, ctype string, url strin
 }
 
 // CreateFromDir create new component with source from local directory
-func CreateFromDir(client *occlient.Client, name string, ctype string, dir string, applicationName string) error {
+func CreateFromDir(client *occlient.Client, name string, ctype string, dir string, applicationName string, sourceType string) error {
```

Comment:
`CreateFromDir` is a misleading name for this function, as it is no longer creating component only from dir, but also from file. Also, `dir`  parametr should be called something else as it is not always a directory.

Reply:
ping @mik-dass